code/modules/reagents/reagents/medical.dm

```diff
@@ -668,6 +684,7 @@
 	custom_metabolism = REAGENTS_METABOLISM * 0.25
 
 /datum/reagent/medicine/quickclot/on_mob_life(mob/living/L, metabolism)
+	L.blood_volume += 0.2
```

Comment:
why?

Reply:
lar Quick Clot buffed slightly. 0.2 bloodheal/tick added. Unit for unit, it's as good as iron, but takes 4 times as long. One pill will restore ~20% of your blood, but will take about 7 minutes. But hey, no sid

Reply:
not a justification

Reply:
I changed it cause I wanted to.

You want me to change it back, gimme a reason to, and we can discuss that.